inc/storefront-template-functions.php

```diff
@@ -183,8 +183,8 @@ function storefront_site_branding() {
 	 */
 	function storefront_primary_navigation() {
 		?>
-		<nav id="site-navigation" class="main-navigation" role="navigation" aria-label="<?php esc_html_e( 'Primary Navigation', 'storefront' ); ?>">
-		<button class="menu-toggle" aria-controls="primary-navigation" aria-expanded="false"><span><?php echo esc_attr( apply_filters( 'storefront_menu_toggle_text', __( 'Menu', 'storefront' ) ) ); ?></span></button>
+		<div id="site-navigation" class="main-navigation" role="navigation" aria-label="<?php esc_html_e( 'Primary Navigation', 'storefront' ); ?>">
```

Comment:
@opportus Seems strange to me to _not_ use the nav tag here. Won't that be harmful? If the lack of header is the problem wouldn't it be better to simply add a header with the `screen-reader-text` class?